contributing.md

```diff
@@ -0,0 +1,106 @@
+# Enarx RFCs
+
+## Contributing
+
+### Do you need an RFC?
+
+Use an RFC to advocate substantial changes to the Aries ecosystem, where
+those changes need to be understood by developers who *use* Aries. Minor
+changes are not RFC-worthy, and changes that are internal in nature,
+invisible to those consuming Aries, should be documented elsewhere.
+
+### Preparation
+
+Before writing an RFC, consider exploring the idea on the
+[chat](https://gitter.im/enarx/community), on a daily meeting call
+(see the [wiki page on contributing](
+https://github.com/enarx/enarx/wiki/How-to-contribute),
+Encouraging feedback from maintainers is a good sign that you're on
+the right track.
+
+### How to propose an RFC
+
+  - Fork [the RFC repo](https://github.com/enarx/rfc).
```

Comment:
~~https://github.com/enarx/rfc~~ -> https://github.com/enarx/RFCs

(the former is 404 Not Found).

Reply:
@npmccallum requested we rename the repo to `rfc` while we were discussing this PR on Gitter. This URL was to reflect that immediately, with plans to change the repo's URL after the PR went through :) I'm switching it to `enarx/rfcs` to align with [rust-lang/rfcs](https://github.com/rust-lang/rfcs/) and keep it all in lowercase.